lib/components/conversationpage/conversationpage.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect, useRef } from "react";
+import axios from "axios";
+import { Card } from "@/components/card";
+import { Modal } from "@/components/modal";
+import { Navbar } from "../navbar";
+
+interface Conversation {
+  id: number;
+  name: string;
+  description: string;
+}
+
+const ConversationPage: React.FC = () => {
+  const [conversations, setConversations] = useState<Conversation[]>([]);
+  const [showModal, setShowModal] = useState(false);
+  const conversationsRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    // Fetch conversations from backend when component mounts
+    const fetchConversations = async () => {
+      try {
+        const response = await axios.get("/api/conversations");
+        setConversations(response.data);
+      } catch (error) {
+        console.error("Error fetching conversations:", error);
+      }
+    };
+
+    fetchConversations();
+  }, []);
+
+  const addConversation = async (name: string, description: string) => {
```

Comment:
Remember to adapt to what I told you about the backend:

We do not have "conversation description", so your modal shouldn't contain this